pkgs/by-name/de/deno/package.nix

```diff
@@ -46,6 +48,8 @@ rustPlatform.buildRustPackage rec {
     cmake
     # required by deno_kv crate
     protobuf
+    # required by libsqlite3-sys
+    sqlite.dev
```

Comment:
This should go to `buildInputs`. `nativeBuildInputs` is for executables required during the build phase